cdap-ui/app/cdap/components/PostRunActions/PostRunActions.scss

```diff
@@ -85,30 +87,95 @@ $fourth-column-width: 53px;
 
 /* Only applying the styles to the post run actions modal in detail view for
 now, since we haven't migrated the one in Studio to use the new component yet */
-.state-hydrator-detail {
-  .modal.hydrator-modal.post-run-actions-modal {
-    .modal-dialog {
-      .modal-content {
-        .modal-body {
-          height: 70vh;
-          .postrun-configuration-content {
-            height: calc(100% - 35px);
-          }
-          .confirm-step-content {
-            .control-label {
-              display: flex;
-
-              .icon-asterisk {
-                top: 5px;
-              }
-            }
-          }
-        }
-        .modal-footer {
-          padding-bottom: 0;
-          border-top: 1px solid #cccccc;
+.post-run-actions-modal.cdap-modal.modal-dialog {
```

Comment:
Seems like a lot of styles are duplicated here from older sass framework. Can we make them css-in-js if we are converting everything? We can have a modal style (not necessarily a modal component) and use them in the post run actions modal?